lib/ffmpeg/transcoder.rb

```diff
@@ -29,19 +29,16 @@ def initialize(movie, output_file, options = EncodingOptions.new, transcoder_opt
       @output_file = output_file
       @transcoder_options = transcoder_options
 
+      self.ensure_temp_directory
+
       # If the movie has frames with varying resolutions, we need to pre-encode the movie for trims
       # This is because ffmpeg can't reliably handle inputs that contain frames with differing resolutions particularly for trimming with `filter_complex`
       @movie.check_frame_resolutions if @transcoder_options[:permit_dynamic_resolution_pre_encode]
 
       if requires_pre_encode
         @movie.paths.each do |path|
           # Make the interim path folder if it doesn't exist
-          dirname = "#{File.dirname(path)}/interim"
```

Comment:
I know this change was my suggestion, but I wonder if we should keep the dirname portion of the path with the temp folder

Reply:
Do you mean we should associate the `dirname = /tmp/interim`?

Reply:
More along the lines of `/tmp/rlovelett/#{File.dirname(path)}/interim`